### PR TITLE
Pagination! 

### PR DESCRIPTION
**Pull request by Goose-Of-War**
Here's to you @nishkalprakash

As of now, it's just the structure and functionality. The styling part is completely left ;-;

### CODE REVIEW

**Review by nishkalprakash — COMMENTED** [review]
awesome!
👍🏻

**Comment by Goose-Of-War** [comment]
@nishkalprakash now?

**Comment by nishkalprakash** [comment]
few things:
- can we add a js to focus on the newsletter div everytime we click on next? (i tried to use arrows after clicking next, didnt work)
- plz yeet the share button or place it elsewhere, it looks horrible
- at the end or start of the newsletter can we dim/disable the button line the next and prev newsletter button
- speaking of... please rename the text of next newsletter to something else

**Comment by Goose-Of-War** [comment]
> few things:
> 
> * can we add a js to focus on the newsletter div everytime we click on next? (i tried to use arrows after clicking next, didnt work)
> * plz yeet the share button or place it elsewhere, it looks horrible
> * at the end or start of the newsletter can we dim/disable the button line the next and prev newsletter button
> * speaking of... please rename the text of next newsletter to something else

* done
* yeet
* done
* done... _something_

**Comment by Goose-Of-War** [comment]
@nishkalprakash ;-;

**Comment by nishkalprakash** [comment]
Great Job

File: templates/_newsletter.njk [hunk 1]
```diff
@@ -3,18 +3,29 @@
...
 
-{% macro nextprev() %}
+{% set pagecount = pages.length %}
+
+
+
+{% macro nextprevnewsletter() %}
 	<div class="controls">
-		<a{% if adjs[0] %} href="/newsletters/{{ adjs[0] }}"{% endif %}><button class="rel" {{ 'disabled' if not adjs[0] }}><span class="buttonicon" id="flipped">&#x279C;</span><span class="buttontext">Previous</span></button></a>
+		<a{% if adjs[0] %} href="/newsletters/{{ adjs[0] }}"{% endif %}><button class="rel" {{ 'disabled' if not adjs[0] }}><span class="buttonicon">&#x21B0;</span><span class="buttontext">Prev Newsletter</span></button></a>
```
Comment (nishkalprakash) [inline templates/_newsletter.njk:12]: I was hoping to change the wordy "prev/next newsletter' to some icon or something, 
but for now its ok

**Review by nishkalprakash — APPROVED** [review]

**Pull request merged**